src/test/java/com/github/martapietka/ppm2pbmSpring/Ppm2PbmControllerTest.java

```diff
@@ -31,18 +34,28 @@ class Ppm2PbmControllerTest {
     @MockBean
     private ImageService imageService;
 
+    @Captor
+    private ArgumentCaptor<InputStream> inputStreamArgumentCaptor;
+
     @Test
     public void shouldReturnCorrectImage() throws Exception {
 
         String imageName = "3x3.ppm";
-        byte[] expectedBytes = Files.readAllBytes(Paths.get("src/test/resources/" + imageName));
-        String expectedContentType = "image/x-portable-graymap";
+        String expectedContentType = "image/png";
+
+        try (InputStream inputStream = getClass().getResourceAsStream("/" + imageName)) {
+
+            byte[] expectedBytes = inputStream.readAllBytes();
+
```

Comment:
Już tutaj można zamknąć try-with-resources, nie ma potrzeby zamykania w bloku całego testu, tylko zaciemnia. Potrzebujesz tylko wczytać tę tablicę bajtów do późniejszego użycia. Jeśli ją zadeklarujesz przed blokiem, a przypiszesz w jego wnętrzu, to będzie nadal dostępna.